operations/mimir-tests/test-ingest-storage-migration-step-8.jsonnet

```diff
@@ -0,0 +1,12 @@
+// Migration step 8:
```

Comment:
Looking at diff for generated YAML files between step 7 and 8, I think we're missing setup of `ingester-zone-[abc]`, before we scale down `ingester-zone-[abc]-partition`. Ingesters without the suffix were removed in step 6, but they still don't exist at this point. Is that OK? If we scale down -partition ingesters now, we have no ingesters running and read-path is broken. WDYT?

Reply:
> I think we're missing setup of ingester-zone-[abc], before we scale down ingester-zone-[abc]-partition

We don't setup `ingester-zone-[abc]` before scaling down `ingester-zone-[abc]-partition` but after that because we'll reuse PVCs (since migrating PVCs is a migration step not done through jsonnet it's not visibile in the codified steps). `ingester-zone-[abc]` is setup in step 9. To keep the steps easier here, I've codified all zones at once, but in practice the migration will be 1 zone at a time to have no downtime.

Reply:
> If we scale down -partition ingesters now, we have no ingesters running and read-path is broken

From step 8:

```
// Migration step 8:
// - Begin migrating ingester-zone-[abc]-partition StatefulSets to ingester-zone-[abc]. This should be done zone-by-zone,
//   but to keep the jsonnet test smaller we're going to do all zones at once. In this step we're going to scale down
//   ingester-zone-[abc]-partition replicas to 0.
```

In practice will be done 1 zone at a time.